### ISSUE TITLE
Am node/javascript guy how can i help

### ISSUE THREAD

**Issue by francishero** [Chizzoz/e-mpiya-foundation#15]
Looks like a good project

**Comment by Chizzoz** [comment]
Hey Francis! If you're familiar with creating GET and POST routes or url endpoints using nodejs and JavaScript, you can create a JavaScript client with the same concept I shared and we could include it as an alternative client in the repository, parallel to the Laravel one. 

But anyhow, if you are still not very familiar with how to go about it in JavaScript, best I can do is implement a parallel client that uses VueJS, let me commit to implementing that, since VueJS will be easy to add on top of this. Also, plain JavaScript can also work, using AJAX.

Basically, you can use any language to implement this, even like an Android App client. So, anyone is free to contribute with a different language or platform to accommodate and help more developers.

Let me know what you think about that. I will also keep this issue alive for VueJS implementation and possibly Android App. Will commit to those two.

**Comment by francishero** [comment]
cool, let me look at the whole project properly. Thanks
